sdk/ai/ai-projects/src/agents/runs.ts

```diff
@@ -108,10 +119,92 @@ export async function createThreadAndRun(
   context: Client,
   options: CreateThreadAndRunParameters,
 ): Promise<ThreadRunOutput> {
+  validateCreateThreadAndRunParameters(options);
   options.body.stream = false;
   const result = await context.path("/threads/runs").post(options);
   if (!expectedStatuses.includes(result.status)) {
     throw createRestError(result);
   }
   return result.body;
 }
+
+function validateThreadId(threadId: string): void {
```

Comment:
Maybe we should move the validateThreadId and validateRunId to the inputValidation file since I think they are/will be needed for other validations (I think other wrapper methods need threadId and runId in the paths)?

Reply:
Good idea, I'll move them into inputValidation